R/as-data-frame.R

```diff
@@ -192,3 +192,48 @@ as.data.frame.FilterCatalog <- function (x,
                                          ...) {
     catalogToDataFrame(x, keys = keys, row.names = row.names, ...)
 }
+
+#' Head and tail methods for Crunch objects. See [utils::head()] for more details.
+#'
+#' @param x a CrunchDataset, CrunchDataFrame, or CrunchVariable
+#' @param n a single integer representing the length of the returning object.
+#' @param ... ignored
+#' @name head-tail
+#' @aliases head tail
+NULL
+
+#' @rdname head-tail
```

Comment:
Since the `head`/`tail` methods that we are imitating are based on S3 classes, we should probably match those and define this (and similar for the following method definitions) as:

`head.CrunchDataset <- function ...`

Then remove the lines in AllGenerics.R that make the generics. This will prevent masking when the package is loaded.